src/components/Form/Autocomplete.js

```diff
@@ -0,0 +1,76 @@
+import React from 'react'
+import Downshift from 'downshift'
+import PropTypes from 'prop-types'
+
+import { ItemsContainer, Items, Inner, styles, itemToString } from './VirtualDropdown'
+import Field from './Field'
+
+const Autocomplete = ({
+  items,
+  label,
+  value,
+  onChange,
+  filter,
+  onFilterChange
+}) => {
+  return (
+    <Downshift {...{
+      onChange,
+      selectedItem: value,
+      onInputValueChange: nextFilter => onFilterChange(nextFilter || ''),
+      itemToString
+    }}>
+      {({
+        getInputProps,
+        getItemProps,
+        getLabelProps,
+        selectedItem,
+        highlightedIndex,
+        isOpen
+      }) => {
+        return <div {...styles.root}>
+          <Inner isOpen={isOpen}>
+            <Field
+              label={label}
+              renderInput={
+                fieldProps => (
+                  <input {...{...fieldProps, ...getInputProps()}} />
```

Comment:
currently focus and blur are not propagated to `<Field />` which leads to incorrect label states.

Could you forward them by doing `{...getInputProps(fieldProps)}`—see [downshift readme](https://github.com/paypal/downshift#getinputprops)—or is there a reason why not?

Incorrect states after blur:
<img width="764" alt="screen shot 2017-11-22 at 20 15 21" src="https://user-images.githubusercontent.com/410211/33145754-ed53a80a-cfc1-11e7-8182-ee76505c4706.png">
Steps: Click field, click else where
Expected: Label animated back to placeholder look

<img width="764" alt="screen shot 2017-11-22 at 20 16 34" src="https://user-images.githubusercontent.com/410211/33145795-11530b74-cfc2-11e7-9089-87cfb668abf6.png">
Steps: Click field, select Januar, click else where
Expected: Label turns grey